src/PhpPact/Consumer/Model/Interaction/IdTrait.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace PhpPact\Consumer\Model\Interaction;
+
+trait IdTrait
+{
+    private int $id;
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
```

Comment:
is there a reason that the trait needs to store as int? I feel like IntIdTrait makes most sense here, as most synthetic primary keys I use are not integers at all.

Reply:
Renamed `id` to `handle`, as how it's called in Rust